addons/website_project_issue/models/project_issue.py

```diff
@@ -1,13 +1,29 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import api, exceptions, models, _
+from odoo import api, fields, exceptions, models
 
 
 class Issue(models.Model):
     _name = "project.issue"
     _inherit = ['project.issue']
 
+    main_attachment_ids = fields.One2many('ir.attachment', compute='_compute_attachment_ids', string="Main Attachments",
+                                          help="Attachment that don't come from message.")
+    website_url = fields.Char('Website URL', compute='_compute_website_url', help='The full URL to access the document through the website.')
+
+    def _compute_website_url(self):
```

Comment:
Every model can have his own route, so each model need to declare his own website_url.

The method name, and field name are not random, it is the same name same website.pblisher mixin. Like that if one day we decide to inherit from it, you don't need to migrate anywthing just remove the field declaration.